app/audit/models/ExtendedAuditModel.scala

```diff
@@ -1,4 +1,4 @@
-@*
+/*
```

Comment:
Hi Miranda, is the thinking behind removing "back_to_business_details" helper because we'll only need it once?

Reply:
@sraspin Yes :)